pkg/hookbot/hookbot.go

```diff
@@ -308,3 +308,19 @@ func (h *Hookbot) ServeSubscribe(conn *websocket.Conn, r *http.Request) {
 		}
 	}
 }
+
+func IsConnectionClose(err error) bool {
```

Comment:
[Yes, well](http://bravenewgeek.com/go-is-unapologetically-flaawed-heres-why-we-use-it/). :dancer: